tests/aws/services/kinesis/test_kinesis.snapshot.json

```diff
@@ -220,5 +220,9 @@
         }
       ]
     }
+  },
+  "tests/aws/services/kinesis/test_kinesis.py::TestKinesis::test_cbor_blob_handling": {
+    "recorded-date": "31-07-2024, 11:17:28",
+    "recorded-content": {}
```

Comment:
nit: seems like this test was run with snapshot update but does not have anything to snapshot, this should be deleted

Reply:
Good catch, thanks! I removed this addition in the latest commit.